api/can_api_v2_definition.py

```diff
@@ -227,70 +235,7 @@ class FieldAnnotations(base_model.APIBaseModel):
-    cases: Optional[FieldAnnotations] = pydantic.Field(None, description="Annotations for cases")
-    deaths: Optional[FieldAnnotations] = pydantic.Field(None, description="Annotations for deaths")
-    positiveTests: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for positiveTests"
-    )
-    negativeTests: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for negativeTests"
-    )
-    contactTracers: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for contactTracers"
-    )
-    hospitalBeds: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for hospitalBeds"
-    )
-    icuBeds: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for icuBeds"
-    )
-    newCases: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for newCases"
-    )
-    newDeaths: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for newDeaths"
-    )
-    vaccinesDistributed: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for vaccinesDistributed"
-    )
-    vaccinationsInitiated: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for vaccinationsInitiated"
-    )
-    vaccinationsCompleted: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for vaccinationsCompleted"
-    )
-    # Keep this list of fields in sync with the fields in `Metrics`
-    testPositivityRatio: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for testPositivityRatio"
-    )
-    caseDensity: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for caseDensity"
-    )
-    contactTracerCapacityRatio: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for contactTracerCapacityRatio"
-    )
-    infectionRate: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for infectionRate"
-    )
-    infectionRateCI90: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for infectionRateCI90"
-    )
-    icuHeadroomRatio: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for icuHeadroomRatio"
-    )
-    icuCapacityRatio: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for icuCapacityRatio"
-    )
-    vaccinationsInitiatedRatio: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for vaccinationsInitiatedRatio"
-    )
-    vaccinationsCompletedRatio: Optional[FieldAnnotations] = pydantic.Field(
-        None, description="Annotations for vaccinationsCompletedRatio"
-    )
+Annotations = NewType("Annotations", Dict[str, Optional[FieldAnnotations]])
```

Comment:
I thought we had the explicit fields in Annotations, while kind of annoying to maintain, to make the JSON structure documented. Why change it?

Reply:
So the main reason why I was playing around with changing it here was because I wasn't totally sure how I was going to represent the source annotations in the API response, so I was thinking this would be a string `vaccinesAdministeredDemographics.age`.  But I might just be able to lump them all together for `vaccinesAdministeredDemographics` as they should be the same

for instance, "vaccinesAdministeredDemographics.age"